osm-georeferencing/main.py

```diff
@@ -82,38 +94,43 @@ def map_to_annotation(specimen_data: Dict, result: Dict[str, Any], timestamp: st
         point_coordinate = result['osm_result']['geometry']
     else:
         point_coordinate = {
-           'coordinates': [
-               result['geopick_result']['decimalLongitude'],
-               result['geopick_result']['decimalLatitude']
-           ]
+            'coordinates': [
+                result['geopick_result']['decimalLongitude'],
+                result['geopick_result']['decimalLatitude']
+            ]
         }
     oa_value = {
-        "georeference": {
-            "dwc:decimalLatitude": round(point_coordinate['coordinates'][1], 7),
-            "dwc:decimalLongitude": round(point_coordinate['coordinates'][0], 7),
-            "dwc:geodeticDatum": 'epsg:4326',
-            "dwc:coordinateUncertaintyInMeters": None if result['is_point'] else
-            result['geopick_result']['coordinateUncertaintyInMeters'],
-            "dwc:pointRadiusSpatialFit": None if result['is_point'] else result['geopick_result']['pointRadiusSpatialFit'],
-            "dwc:coordinatePrecision": 0.0000001,
-            "dwc:footprintSRS": 'epsg:4326',
-            "dwc:footprintWKT": from_geojson(json.dumps(result.get('osm_result').get('geometry'))).wkt,
-            "dwc:footprintSpatialFit": None if result['is_point'] else 1,
-            "dwc:georeferencedBy": f"https://hdl.handle.net/{os.environ.get('MAS_ID')}",
-            "dwc:georeferencedDate": timestamp,
-            "dwc:georeferenceSources": 'GeoPick v.1.0.4',
-            "dwc:georeferenceProtocol": 'Georeferencing Quick Reference Guide (Zermoglio et al. 2020, '
-                                        'https://doi.org/10.35035/e09p-h128)',
-            "dwc:georeferenceRemarks": f"This georeference was created by the GeoPick API. Based on OpenStreetMap API "
-                                       f"query of {result['queryString']}"
-        }
+        shared.AT_TYPE: "ods:GeoReference",
+        "dwc:decimalLatitude": round(point_coordinate['coordinates'][1], 7),
+        "dwc:decimalLongitude": round(point_coordinate['coordinates'][0],
+                                      7),
+        "dwc:geodeticDatum": 'epsg:4326',
+        "dwc:coordinateUncertaintyInMeters": None if result['is_point'] else
+        result['geopick_result']['coordinateUncertaintyInMeters'],
+        "dwc:pointRadiusSpatialFit": None if result['is_point'] else
+        result['geopick_result']['pointRadiusSpatialFit'],
+        "dwc:coordinatePrecision": 0.0000001,
+        "dwc:footprintSRS": 'epsg:4326',
+        "dwc:footprintWKT": from_geojson(
+            json.dumps(result.get('osm_result').get('geometry'))).wkt,
+        "dwc:footprintSpatialFit": None if result['is_point'] else 1,
+        "dwc:georeferencedBy": f"https://hdl.handle.net/{os.environ.get('MAS_ID')}",
+        "dwc:georeferencedDate": timestamp,
+        "dwc:georeferenceSources": 'GeoPick v.1.0.4',
```

Comment:
They went to a 2.X version recently, do we have a task for this or should this one work with the latest version?

Reply:
https://naturalis.atlassian.net/browse/DD-1346?atlOrigin=eyJpIjoiNWNhNWFlY2I2NDI5NDIwNjhhYzZiZTU4YjBjZWQ1OWIiLCJwIjoiaiJ9